### PR TITLE
update ospo associate info

### PR DESCRIPTION
**Pull request by anajsana**
Signed-off-by: Ana Jimenez Santamaria <ana@todogroup.org>

### CODE REVIEW

File: OSPO-Associate-Program.md [hunk 1]
```diff
@@ -2,13 +2,19 @@
 
 ## What is the OSPO Associate Program?
 
+TODO Group Community is open to all. Sometimes, there are community participants who wish show their organization efforts on fostering OSPO adoption and enable the worldwide community of practitioners.
```
Comment (tsteenbe) [inline OSPO-Associate-Program.md:5]: Community -> community

File: OSPO-Associate-Program.md [hunk 2]
```diff
@@ -2,13 +2,19 @@
...
+
+Community participants comming from and organization with an established OSPO or similar open source initiative can support TODO becoming `General Members`. To those open source communities or project focused on [OSPO-related topics](https://ospomindmap.todogroup.org/) who help OSPOs but may not have an OSPO, they can support TODO becoming `OSPO Associates`.
```
Comment (tsteenbe) [inline OSPO-Associate-Program.md:7]: comming -> coming

Comment (tsteenbe) [inline OSPO-Associate-Program.md:7]: becoming `General Members`. -> by becoming `General Members`.

Comment (tsteenbe) [inline OSPO-Associate-Program.md:7]: and organization -> an organization

**Comment by anajsana** [comment]
@tsteenbe @vmbrasseur @ashleywolf @geekygirldawn I've made some edits to my PR to clarify who can qualify for OSPO Associate, its benefits, and how to apply based on your comments from [the SC slack channel conversation](https://thetodogroup.slack.com/archives/C02RLEUHBTP/p1655240557882309). 

Please review this when you have time 👍 Thank you!

File: OSPO-Associate-Program.md [hunk 3]
```diff
@@ -1,29 +1,55 @@
...
+
+## 🚀 What are the benefits of becoming an OSPO Associate?
 
 Benefits include:
 
 * TODO OSPO Associate Badge
-* Appear at OSPO Landscape as OSPO Associate
-* Promotion at TODO Website and main communication channels
+* Appear at OSPO Landscape as [OSPO Associate](https://landscape.todogroup.org/card-mode?category=ospo-associate&grouping=category)
+* Promotion at TODO Website and main communication channels (TODO mailing lists, newsletter and social media channels)
```
Comment (tsteenbe) [inline OSPO-Associate-Program.md:45]: Website -> website

File: OSPO-Associate-Program.md [hunk 4]
```diff
@@ -1,29 +1,55 @@
 ![ospoassocaite_program](https://user-images.githubusercontent.com/43671777/139878216-97c4ac50-d3c3-476a-a63c-2a0aeb15b12e.png)
 
-## What is the OSPO Associate Program?
+## 💚 Ways to support TODO as an entity
 
-OSPO Associates is a TODO Program that aims to work collaboratively across communities that are helping the OSPO movement to build the most comprehensive OSPO solutions infrastructure and network.
+TODO Group community is open to all. Sometimes, there are community participants who wish show their organization efforts on fostering OSPO adoption and enable the worldwide community of practitioners.
```
[the remark this answers is not in the document]
Comment (tsteenbe) [reply OSPO-Associate-Program.md:5]: "TODO Group community is open to all." propose we more clear about community vs. members

**Review by tsteenbe — CHANGES_REQUESTED** [review]

File: OSPO-Associate-Program.md [hunk 5]
```diff
@@ -1,29 +1,55 @@
 ![ospoassocaite_program](https://user-images.githubusercontent.com/43671777/139878216-97c4ac50-d3c3-476a-a63c-2a0aeb15b12e.png)
 
-## What is the OSPO Associate Program?
+## 💚 Ways to support TODO as an entity
```
[the remark this answers is not in the document]
Comment (anajsana) [reply OSPO-Associate-Program.md:3]: Ok! Then I would also mention that individuals can also support TODO by becoming contributors to the different TODO repos (like OSPOlogy)

File: OSPO-Associate-Program.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (anajsana) [reply OSPO-Associate-Program.md:5]: I'll add "TODO Group is formed by its Community participants and General Members"

[the remark this answers is not in the document]
Comment (anajsana) [reply OSPO-Associate-Program.md]: @tsteenbe I'm trying to make clear that people can contribute to TODO resources and participate in TODO sync calls as individuals without applying for GM or Associates program as an organization. That's why "TODO Community is open to all"

**Comment by anajsana** [comment]
Committed the requested changes from @tsteenbe. I'll merge this PR. Please open a new PR if you'd like to refine this documentation.

**Pull request merged**